bundles/org.openhab.binding.salus/README.md

```diff
@@ -75,11 +75,12 @@ removed.
 
 ### `salus-it600-device` Channels
 
-| Channel                     | Type               | Read/Write | Description                                                                                                                                                                                                                                                                                     |
-|-----------------------------|--------------------|------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| it600-temp-channel          | Number:Temperature | RO         | Current temperature in the room                                                                                                                                                                                                                                                                 |
+| Channel                    | Type               | Read/Write | Description                                                                                                                                                                                                                                                                                     |
+|----------------------------|--------------------|------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| it600-temp-channel         | Number:Temperature | RO         | Current temperature in the room                                                                                                                                                                                                                                                                 |
 | it600-expected-temp-channel | Number:Temperature | RW         | Sets the desired temperature in the room                                                                                                                                                                                                                                                        |
```

Comment:
```suggestion
| expected-temperature | Number:Temperature | RW         | Sets the desired temperature in the room                                                                                                                                                                                                                                                        |
```

According to the it600.xml this channel id is different. Looks like you added the type id, while it should be the channel id here.